webapp/src/components/utils.jsx

```diff
@@ -64,3 +64,9 @@ export const getStyleForReactSelect = (theme) => {
         }),
     };
 };
+
+// TODO: change to use the isCombinedUserActivityPost in 'mattermost-redux/utils/post_list' when we upgrade to 5.12
```

Comment:
(non-blocking) can I get an explanation for this please?

Reply:
@levb I just assumed we should use the original function from mattermost-redux (the one here is a cut and paste). And since it was added pretty recently in mattermost-redux, we probably can't assume an older version has it. That's all. Is there a better way to do this?